src/Pages/Form/Form.css

```diff
@@ -0,0 +1,54 @@
+/* Text box container styles */
+.data-container{
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+  padding-bottom: 60px;
+}
+
+.textbox-container {
+  display: flex; /* Enable flexbox layout */
+  justify-content: center; /* Center horizontally within the container */
+  align-items: center; /* Center vertically within the container */
+  width: 100%; /* Make the container take up the full width */
+}
+
+.underlined{
+  text-decoration: underline;
+}
+
+/* Text box styles */
```

Comment:
Maybe you can put a max-height property instead of putting a fixed height. Lather on we can bind to this max height to know how long is the text.